tests/test_utils.ts

```diff
@@ -427,7 +427,9 @@ async function spawnServer(
   for await (const line of lines.values({ preventCancel: true })) {
     output.push(line);
     const match = line.match(/https?:\/\/localhost:\d+/g);
-    if (match) {
+    const running = line.match("Fresh ready");
+
+    if (running) {
       address = match[0];
```

Comment:
This line is likely causing the tests to fail. Previously the regex was used to grab the actual server address out of the child process stdout and that's missing now.

Reply:
I reverted the change to "Fresh ready" and just made the regex less strict, which should do the trick.